tests/lwip/tests/01-run.py

```diff
@@ -268,6 +269,53 @@ def test_tcpv6_send(board_group, application, env=None):
         client.expect_exact(u"could not send")
 
 
+def test_tcpv6_multiconnect(board_group, application, env=None):
+    if any(b.name != "native" for b in board_group.boards):
+        # run test only with native
+        print("SKIP_TEST INFO found non-native board")
+        return
+    env_client = os.environ.copy()
+    if env is not None:
+        env_client.update(env)
+    env_client.update(board_group.boards[0].to_env())
+    env_server = os.environ.copy()
+    if env is not None:
+        env_server.update(env)
+    env_server.update(board_group.boards[1].to_env())
+    with pexpect.spawnu("make", ["-C", application, "term"], env=env_client,
+                        timeout=DEFAULT_TIMEOUT) as client, \
+        pexpect.spawnu("make", ["-C", application, "term"], env=env_server,
+                       timeout=DEFAULT_TIMEOUT) as server:
+        port = random.randint(0x0000, 0xffff)
+        server_ip = get_ipv6_address(server)
+        client_ip = get_ipv6_address(client)
+
+        try:
+            connect_addr = socket.getaddrinfo(
+                "%s%%tapbr0" % server_ip, port)[0][4]
+        except socket.gaierror as e:
+            print("SKIP_TEST INFO", e)
+            return
```

Comment:
These tests need to be adapted for `riotnode` (#10949) once that is merged anyway, so I hope you don't mind `tapbr0` being statically used here and the test being skipped instead as a short workaround. Otherwise I would need a third "board" just for this test.